example/install.sh

```diff
@@ -0,0 +1,19 @@
+rm -rf ./AwesomeProject
+react-native init AwesomeProject
```

Comment:
What's the point of this script? add a comment

Reply:
You asked for a script to auto install the example app, this is it.